### PR TITLE
improve TextField example screen

### PR DESCRIPTION
**Pull request by ethanshar**
I separated the text field screen to 3 
- General Inputs - what we have today
- Custom Inputs - showing different custom UI styling 
- Input Validation - show the (new) validation functionality in depth 

This is just the initial work, but from here we can start re-arrange the input examples better. 
It will also help when we'll add e2e tests

### CODE REVIEW

File: demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js [hunk 1]
```diff
@@ -41,22 +41,22 @@ export default class InputsScreen extends Component {
       message = 'Please enter a valid text';
     }
     this.setState({error: message});
```
Comment (Inbal-Tish) [inline demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js]: If there is a dedicated screen for validations why not clean this screen from all error examples to keep it simple?

File: demo/src/screens/componentScreens/TextFieldScreen/InputValidationsScreen.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React, {Component} from 'react';
+import {StyleSheet, ScrollView} from 'react-native';
+import {View, Text, TextField, Spacings} from 'react-native-ui-lib';
+import {Navigation} from 'react-native-navigation';
+import {KeyboardAwareInsetsView} from 'react-native-keyboard-tracking-view';
+
+class InputValidationsScreen extends Component {
```
Comment (Inbal-Tish) [inline demo/src/screens/componentScreens/TextFieldScreen/InputValidationsScreen.js]: Don't you want to add an `useTopErrors` example? Maybe a toggle button to switch between the top and the bottom error like in the input screen?

File: demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js (hunk 1 above)
Comment (ethanshar) [reply demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js]: yes, that's the plan.. that what I will do afterwards.. 
I just wanted to start simple and create the different screens without changing the current one.. I plan to clean the general screen (it has too many examples)

Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js]: Cool. So you want to merge it like that or after the cleanup?

Comment (ethanshar) [reply demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js]: Done. I'll make it more simple in the future, for now I move the error/validation examples

File: demo/src/screens/componentScreens/TextFieldScreen/InputValidationsScreen.js (hunk 2 above)
Comment (ethanshar) [reply demo/src/screens/componentScreens/TextFieldScreen/InputValidationsScreen.js]: done

**Pull request merged**